src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+import { Trans } from 'react-i18next';
+import { Link } from 'react-router-dom'
+
+import {
+  Flyout,
+  FlyoutHeader,
+  FlyoutTitle,
+  FlyoutCloseBtn,
+  FlyoutContent,
+} from 'components/shared';
+
+import './SIMManagement.css';
+
+export class SIMManagement extends Component {
```

Comment:
Unless this component is going to become more complicated in the near future this is a good place for a functional component.

```
export const SIMManagement = ({ t, onClose }) => (
  <Flyout>
    ...
  </Flyout>
);
```